fio/fio_run

```diff
@@ -1083,6 +1084,10 @@ if [ $file_count -eq 0 ]; then
 			if [ $? -ne 0 ]; then
 				exit_out "Error: $item is not a block device" 1
 			fi
+			test_tools/detect_mounts $item
+			if [ $? -ne 0 ]; then
+				exit_out "Error: $item is mounted." 1
+			fi
```

Comment:
To make @frival happy, this seems to add another error check separate to the pbench runs fix.

Open a separate PR for this stuff.

Reply:
In this case I won't complain too loudly *iff* this change is a separate commit and commits aren't squashed.  This is your benevolent dictator, signing off.

Reply:
Moved to own pr.